fuse_core/include/fuse_core/graph.h

```diff
@@ -381,7 +381,8 @@ class Graph
    */
   virtual ceres::Solver::Summary optimizeFor(
     const rclcpp::Duration& max_optimization_time,
-    const ceres::Solver::Options& options = ceres::Solver::Options()) = 0;
+    const ceres::Solver::Options& options = ceres::Solver::Options(),
+    rclcpp::Clock clock = rclcpp::Clock(RCL_STEADY_TIME)) = 0;  // NOTE(CH3): We need to copy because clock.now() is non-const...
```

Comment:
Opened PR to make `rclcpp::Clock::now()` const: https://github.com/ros2/rclcpp/pull/2050